feature-group-definitions/text-spacing-trim.yml

```diff
@@ -2,5 +2,3 @@ name: text-spacing-trim
 description: The `text-spacing-trim` CSS property controls spacing around CJK characters, avoiding excessive whitespace when using full-width punctuation characters.
 spec: https://drafts.csswg.org/css-text-4/#text-spacing-trim-property
 usage_stats: https://chromestatus.com/metrics/css/timeline/popularity/771
-compat_features:
-  - css.properties.text-spacing-trim
```

Comment:
I took the liberty of dropping this since I was refreshing this dist anyway.

Reply:
That's what I would do too.